### PR TITLE
Table creation

### PR DESCRIPTION
**Pull request by jasminejanet**
kindly review my table program..

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <script src="lib/babel.min.js"></script>
+    <script src="lib/react.min.js"></script>
+    <script src="lib/react-dom.min.js"></script>
+      <script type="text/javascript" src="lib/JSXTransformer.js"></script> 
+      <script type="text/javascript" src="lib/react-with-addons.js"></script>
+      <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+  </head>
+  <body>
+    <div id="jas"></div>
+    <script type="text/jsx;harmony=true" src="scripts/type3.js"></script>
```
Comment (Sharavanth) [inline public/index.html]: The file could be named better , eg table.js

Comment (Sharavanth) [reply public/index.html]: Also unneccesary files could be removed

File: public/scripts/type3.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    },
+
+    generateRows: function() {
+        var cols = this.props.cols, 
+            data = this.props.data;
+
+        return data.map(function(item) {
+          
+            var cells = cols.map(function(colData) {
+
```
Comment (Sharavanth) [inline public/scripts/type3.js]: Avoid blank lines. Try to conform to coding style guidelines

Comment (jasminejanet) [reply public/scripts/type3.js]: Yes.. I have removed now

File: public/index.html (hunk 1 above)
Comment (jasminejanet) [reply public/index.html]: ok have done this too...

**Pull request merged**